extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -69,16 +74,26 @@
   background-color: var(--i-amphtml-interactive-theme-border) !important;
 }
 
+.i-amphtml-story-interactive-option
+  .i-amphtml-story-interactive-option-percent-bar {
+  height: 100%;
+  width: 100%;
+  position: absolute;
+  opacity: 0;
+  transform-origin: left;
```

Comment:
Do you need to override this for RTL languages?

Reply:
Yes! Great catch. Thank you.